lib/rspec/core/output_wrapper.rb

```diff
@@ -3,17 +3,25 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        output.respond_to?(name, priv)
+      end
+
+      def method_missing(name, *args, &block)
+        output.method_missing(name, *args, &block)
+      end
```

Comment:
I understand why this is needed now.  But I think it's undesirable to have an `OutputWrapper` class that we specifically have designed to wrap non-output objects.  It's an _output_ wrapper, so it's far less confusing to limit it to only wrapping output objects, and then we don't have to mess with this `respond_to?` and `method_missing` weirdness.

Here's one way to do it:

046179ca43bf96964a51ff645a5007d650e6c742

That passes locally for me.